### PR TITLE
187936148 map layer palette tuneup

### PR DESCRIPTION
**Pull request by eireland**
Restyles the map tool palettes.

### CODE REVIEW

File: v3/src/components/inspector-panel.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@ $inspector-panel-width: 50px;
   top: 0;
   width: $inspector-panel-width;
   min-height: 50px;
-  z-index: 10;  // Will need to be higher to accommodate Leaflet
+  z-index: 50;
```
Comment (kswenson) [inline v3/src/components/inspector-panel.scss:10]: Out of curiosity, what was the symptom that led you to bump up the `z-index` here? I'd like to consider whether there might be a better solution than `z-index` inflation.

Comment (eireland) [reply v3/src/components/inspector-panel.scss:10]: This was mainly for the window expand/contract icon to not appear on top of the palette.

Comment (kswenson) [reply v3/src/components/inspector-panel.scss:10]: There may be a better solution lurking here, but for now make it `15` and we can put off a deeper analysis for later.
```suggestion
  z-index: 15;
```

**Review by kswenson — APPROVED** [review]
👍 LGTM -- just one suggestion on `z-index`.

**Pull request merged**